custom_news_feed/custom_news_feed.py

```diff
@@ -266,13 +271,20 @@ def display_news_content(self, news_json_file_path):
             self.dockwidget.tabWidget.setTabText(1, news['PanelTitleFeedRepository'])
             self.dockwidget.linkSectionLabel.setText(news['LinkSectionTitle'])
             self.settings_dlg.pathToConfigurationFileLabel.setText(news["PathToConfigurationFileLabel"])
-            if self.checkPublishingDate(news["PinnedMessage"]['StartPublishingDate'],news["PinnedMessage"]['EndPublishingDate']) == True:
-                self.current_pinned_message = news["PinnedMessage"]
-                self.configure_pinned_message(news["PinnedMessage"])
+
+            self.current_pinned_message = news["PinnedMessage"]
+            if 'StartPublishingDate' in json.loads(json.dumps(news["PinnedMessage"])) and 'EndPublishingDate' in json.loads(json.dumps(news["PinnedMessage"])):
```

Comment:
PP: `json.loads(json.dumps(news["PinnedMessage"])` in Variable auslagern. So wird das Ganze immer nur einmal aufgerufen.